### DISCUSSION TITLE
No progress-bar while building

### DISCUSSION THREAD

**Discussion by tomers** [quasarframework/quasar#10285] [Ideas / Proposals]
While building a Quasar app, I get:
```
Starting type checking service...

 • Compiling:
 └── SPA ███                  12% building 23/31 modules 8 active

=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.1.0

YOUR TYPESCRIPT VERSION: 4.2.2

 • Compiling:
 └── SPA ████                 17% building 65/95 modules 30 active

=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.1.0

YOUR TYPESCRIPT VERSION: 4.2.2

 • Compiling:
 └── SPA ████████████████████ 100% done in 24468 ms


```
Having these progress bars shown in CI is rather verbose and does not supply much information.
In tools like `curl`, `wget`, etc. there is usually a CLI option to turn it off, e.g. `--quiet`. 
This could be useful here.

**Comment by hawkeye64** [comment]
The type of terminal is detected and if it can't handle the progress bar, then it isn't displayed. I agree a `-q` for "quiet" would be a nice-to-have. Maybe ask for it in the "issues" section as a feature.

**Reply by tomers** [reply]
Thanks! I've opened issue #10304.

> The type of terminal is detected and if it can't handle the progress bar, then it isn't displayed

I saw progress bars in my Jenkins run, but it shows a `quasar build` command that was run inside a docker, so it could detect a different terminal. This is the exact reason why such flag is needed.

**Reply by hawkeye64** [reply]
I agree. If it can detect the terminal type and not show it, then having a flag should/could be minimal work.
